### PR TITLE
Admin ajax variations

### PR DESCRIPTION
**Pull request by claudiosanches**
This PR added ajax methods to load, save and delete variations for #6995

Here an example how it works: https://cloudup.com/cjI4t5ryk2i

I tried to change as little as possible, trying to do good part of things only with JavaScript.

I hope I have not broken compatibility with 3rd party plugins (cc: @thenbrent)

### CODE REVIEW

File: assets/js/admin/meta-boxes-product-variation.js [hunk 1]
```diff
@@ -1,424 +1,746 @@
...
+		/**
+		 * Paginav pagination selector
+		 */
+		page_selector: function() {
+			var selected = parseInt( $( this ).val(), 10 ),
+				wrapper  = $( '#variable_product_options .woocommerce_variations' );
 
-		} else {
+			if ( wc_meta_boxes_product_variations_ajax.check_for_editions() ) {
+				console.log( 'page_selector', selected );
```
Comment (thenbrent) [inline assets/js/admin/meta-boxes-product-variation.js]: @claudiosmweb was this intentionally left in? :smile:

**Comment by thenbrent** [comment]
Works perfectly with Variable Subscription products. Thanks! :+1:

File: assets/js/admin/meta-boxes-product-variation.js (hunk 1 above)
Comment (claudiosmweb) [inline assets/js/admin/meta-boxes-product-variation.js]: I forget xD
Thanks

**Comment by claudiosmweb** [comment]
@thenbrent I did not broken anything?

**Comment by thenbrent** [comment]
Haha it isn't working 100% - we'll need to update Subscriptions' JavaScript to show/hide the subscription specific variation fields. But it's working pretty darn well considering how much of a major change it is. :smile:

**Comment by claudiosmweb** [comment]
@thenbrent great!
If you need more triggers, please let me know.

**Pull request merged**